homeassistant/helpers/condition.py

```diff
@@ -56,6 +58,174 @@
 ConditionCheckerType = Callable[[HomeAssistant, TemplateVarsType], bool]
 
 
+# Common helpers for tracing. TODO: Move to own file
+def trace_stack_push(trace_stack_var, node):
+    """Push an element to the top of a trace stack."""
+    trace_stack = trace_stack_var.get()
+    if trace_stack is None:
+        trace_stack_var.set([])
+        trace_stack = trace_stack_var.get()
```

Comment:
```suggestion
        trace_stack = []
        trace_stack_var.set(trace_stack)
```

Reply:
Fixed (but in new file helpers/trace.py)